src/main/java/com/google/sps/servlets/MentorApprovalServlet.java

```diff
@@ -0,0 +1,130 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.classes.MentorEvidence;
+import com.google.sps.classes.SqlConstants;
+import com.google.sps.classes.Utility;
+import java.io.IOException;
+import java.sql.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet that handles mentor evidence for approver to see.
+ */
+@WebServlet("/mentor-approval")
+public class MentorApprovalServlet extends HttpServlet {
+
+  /**
+   * Returns information about a mentor and the evidence of their internship.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+
+    // Get IDs of mentor and approver.
+    int mentorId = Utility.tryParseInt(request.getParameter("id"));
+    int approverId = Utility.getUserId();
+    
+    // Set default variables to create MentorEvidence object.
+    boolean isApprover = false;
+    String mentorUsername = "";
+    String paragraph = "";
+
+    if (userService.isUserLoggedIn()) {
+      // If user is logged in, update variables.
+      isApprover = checkForApprover(mentorId, approverId);
+      mentorUsername = Utility.getUsername(mentorId);
+      paragraph = getMentorEvidence(mentorId);
+    }
+
+    MentorEvidence mentorEvidence =
+        new MentorEvidence(isApprover, mentorUsername, paragraph);
```

Comment:
What does this do if the user is not logged in? Might be worth clarifying with a comment.

Reply:
In theory, that should never happen, but if it does, it just sends default values. I modified the comment on line 53 to say this: `If user is logged in, update variables. Else, variables stay with default values.` Is that good?

Reply:
I guess my question is more what happens downstream from this call? If MentorEvidence is created and passed along with default values, what will be shown to the user? It would be good to clarify that in the comment above as well.